quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
```

Comment:
if this is generated by bicep, include only the bicep source file...

Reply:
This is a Prerequisite file and is not generated by bicep File

Reply:
The metadata property has the bicep header in it... so this file, at some point, was generated by bicep or someone manually authored it to look that way (which would be odd but anyway)...

if there's bicep source for the file add it to the PR and remove the json.  If there's no bicep source, remove the metadata.

Reply:
Removed Metadata